### PR TITLE
Revert "fix: try to remove unnecessary paragraph", which caused #418

### LINKED ISSUE #418: 5.0.0-rc6 sections not properly added to document
In `5.0.0-rc6` sections are not properly being added to the document. This issue is not present on `5.0.0-rc5`.

## Results of test

Below are screenshots of the resulting `.docx` files using `5.0.0-rc5` and `5.0.0-rc6`. Both are generated with the same script. See "Recreating the issue" section below.

### RC5
image

### RC6
image

## Recreating the issue

I used the following script to be able to easily point to two separate installs of DOCX. By running `node lib-docx-test2.js lib-docx-rc5` could run the script using my `lib-docx-rc5` dir, and easily switch to my `lib-docx-rc6` directory by changing the last argument. I confirmed in both of these directories that I had checked out the appropriate git tags and had run `npm install && npm run-script build`.

```javascript
const dir = process.argv[2];
if (!dir) {
	console.log('Need to specify which docx version to use');
	process.exit();
}
const docxPackageJson = require(`./${dir}/package.json`);
const docx = require(`./${dir}/build`);
const fs = require('fs');
const path = require('path');

const doc = new docx.Document();

const header = function(title) {
	return new docx.Header({
		children: [new docx.Paragraph({
			text: `${title} - created with docx@${docxPackageJson.version}`
		})]
	});
};

doc.addSection({
	headers: {
		default: header('section one')
	},
	children: [
		new docx.Paragraph(`section 1 - created with docx@${docxPackageJson.version}`)
	]
});

doc.addSection({
	headers: {
		default: header('section 2')
	},
	children: [
		new docx.Paragraph(`section 2 - created with docx@${docxPackageJson.version}`)
	]
});

const relativeFilepath = path.relative(process.cwd(), `docx-test2-${dir}.docx`);

docx.Packer.toBuffer(doc).then((buffer) => {
	fs.writeFileSync(relativeFilepath, buffer);
});
```

## Difference in underlying XML

Diffing the underlying XML of the `.docx` file version created with `5.0.0-rc5` versus `5.0.0-rc6` I get the following:

```diff
+++ b/sections.docx.contents/word/document.xml
@@ -1,39 +1,18 @@
     <w:body>
-        <w:p/>
         <w:p>
             <w:r>
                 <w:t xml:space="preserve">
-                    section 1 - created with docx@5.0.0-rc5
+                    section 1 - created with docx@5.0.0-rc6
                 </w:t>
             </w:r>
         </w:p>
-        <w:p>
-            <w:pPr>
-                <w:sectPr>
-                    <w:pgSz w:w="11906" w:h="16838" w:orient="portrait"/>
-                    <w:pgMar w:top="1440" w:right="1440" w:bottom="1440" w:left="1440" w:header="708" w:footer="708" w:gutter="0" w:mirrorMargins="false"/>
-                    <w:cols w:space="708" w:num="1"/>
-                    <w:docGrid w:linePitch="360"/>
-                    <w:headerReference w:type="default" r:id="rId5"/>
-                    <w:footerReference w:type="default" r:id="rId6"/>
-                </w:sectPr>
-            </w:pPr>
-        </w:p>
         <w:p>
             <w:r>
                 <w:t xml:space="preserve">
-                    section 2 - created with docx@5.0.0-rc5
+                    section 2 - created with docx@5.0.0-rc6
                 </w:t>
             </w:r>
         </w:p>
-        <w:sectPr>
-            <w:pgSz w:w="11906" w:h="16838" w:orient="portrait"/>
-            <w:pgMar w:top="1440" w:right="1440" w:bottom="1440" w:left="1440" w:header="708" w:footer="708" w:gutter="0" w:mirrorMargins="false"/>
-            <w:cols w:space="708" w:num="1"/>
-            <w:docGrid w:linePitch="360"/>
-            <w:headerReference w:type="default" r:id="rId7"/>
-            <w:footerReference w:type="default" r:id="rId8"/>
-        </w:sectPr>
     </w:body>
 </w:document>
\ No newline at end of file
diff --git a/sections.docx.contents/word/header1.xml b/sections.docx.contents/word/header1.xml
index 9f905ad..578d17f 100644
--- a/sections.docx.contents/word/header1.xml
+++ b/sections.docx.contents/word/header1.xml
@@ -2,7 +2,7 @@
     <w:p>
         <w:r>
             <w:t xml:space="preserve">
-                section one - created with docx@5.0.0-rc5
+                section one - created with docx@5.0.0-rc6
             </w:t>
         </w:r>
     </w:p>
diff --git a/sections.docx.contents/word/header2.xml b/sections.docx.contents/word/header2.xml
index c5707d1..959d2cf 100644
--- a/sections.docx.contents/word/header2.xml
+++ b/sections.docx.contents/word/header2.xml
@@ -2,7 +2,7 @@
     <w:p>
         <w:r>
             <w:t xml:space="preserve">
-                section 2 - created with docx@5.0.0-rc5
+                section 2 - created with docx@5.0.0-rc6
             </w:t>
         </w:r>
     </w:p>
```

### PR DESCRIPTION
**Pull request by jamesmontalvo3**
Fix broken multi-section documents.

- This reverts commit cb7486824764ac25ae5617e1a7028dd957376f02 / PR #391 
- Closes #418 
- Re-opens #388 (I personally had said #388 was resolved several days ago, but must not have tested against a multi-section document. Apologies.)

FYI @bokuweb

### CODE REVIEW

**Comment by jamesmontalvo3** [comment]
@dolanmiu if you agree this is a good revert, could you also publish a new RC to NPM to simplify having a working version?

File: src/file/document/body/body.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ export class Body extends XmlComponent {
      * @param options new section options
      */
     public addSection(options: SectionPropertiesOptions): void {
+        const currentSection = this.sections.pop() as SectionProperties;
```
Comment (dolanmiu) [inline src/file/document/body/body.ts]: im having a hard time why the fact of doing a `.pop` will fix issues

Comment (jamesmontalvo3) [reply src/file/document/body/body.ts]: I’m not sure. I just know that this commit was intended to remove an unnecessary paragraph, which it succeeded at in some cases but in others it has disastrous effects. It makes sense to me to revert it to put the project back into a functioning state. Further optimization can happen after that.

Comment (dolanmiu) [reply src/file/document/body/body.ts]: Good point, will merge

**Comment by jamesmontalvo3** [comment]
Thanks!

**Pull request merged**